app/Http/Controllers/Kits/CheckoutKitController.php

```diff
@@ -4,6 +4,7 @@
 use App\Http\Controllers\CheckInOutRequest;
 use App\Http\Controllers\Controller;
 use App\Models\PredefinedKit;
+use App\Models\Asset;
```

Comment:
This doesn’t seem necessary - I don’t see `Asset` being used later on in this file?

Reply:
We were using the gate check `$this->authorize('checkout', Asset::class);` already, but because we weren't including the `use` statement, Laravel was looking for a gate definition for `\App\Models\PredefinedKits\Asset`, so it would always fail, since that model doesn't even exist. 

https://github.com/snipe/snipe-it/blob/f4650c65db272ee6f724e28dfe62e45ff6f4ff02/app/Http/Controllers/Kits/CheckoutKitController.php#L39

Reply:
That makes perfect sense, thank you. PHP interally processes `blah::class` as “Whatever blah is, but prepend the namespace and treat it as a string” - apparently, in that process, it doesn’t bother to figure out whether or not the thing is an actual Class that exists. Blech.